src/app/core/__init__.py

```diff
@@ -0,0 +1,15 @@
+from functools import lru_cache
+
+from app.core.settings import Settings
+
+
+@lru_cache
+def get_settings() -> Settings:
+    """Instantiating the Settings object using LRU caching
```

Comment:
What does the LRU provide here, and what constitutes a cache hit and a cache miss?

Reply:
This was carried over from a previous template. Removing.